projects/storefrontlib/cms-components/myaccount/consent-management/components/consent-form/consent-management-form.component.html

```diff
@@ -4,7 +4,7 @@
       type="checkbox"
       class="form-check-input"
       (change)="onConsentChange()"
-      [checked]="consentGiven"
```

Comment:
If we add the feature flag to the component file we might need one here too

Reply:
Agree